lib/resolveScripts.js

```diff
@@ -0,0 +1,58 @@
+var path = require('path');
+var semver = require('semver');
+var Handlebars = require('handlebars');
+var versionMap = require('./version-map.json');
+
+var defaultPaths = {
+  'jquery': Handlebars.compile('http://ajax.googleapis.com/ajax/libs/jquery/{{version}}/jquery.min.js'),
+  'can': Handlebars.compile('http://canjs.com/release/{{version}}/can.jquery.js'),
+  'ejs': Handlebars.compile('http://canjs.com/release/{{version}}/can.ejs.js'),
+  'mustache': Handlebars.compile('http://canjs.com/release/{{version}}/can.view.mustache.js'),
```

Comment:
I don't think this is available for all versions. I guess it doesn't matter too much since it is in core already though.